datamodel/extra_scripts/oracle/106_bag_views.sql

```diff
@@ -1,750 +1,385 @@
---
--- Views for visualizing the bag data.
---
+/*
+Views for visualizing the BAG data.
+versie 2
+22-6-2018
+*/
+-- DROP VIEWS
+--drop view v_adres cascade;
```

Comment:
dit is geen geldige oracle sql, in 11.2 is er wel `drop view <view> cascade constraints`